projectgenerator/libqgsprojectgen/dbconnector/db_connector.py

```diff
@@ -31,8 +31,14 @@ def map_data_types(self, data_type):
         '''Map provider date/time types to QGIS date/time types'''
         return None
 
+    def db_or_schema_exists(self):
+        '''Whether the DB (for GPKG) or schema (for PG) exists or not.'''
+        return False
```

Comment:
Should this raise a `NotImplementedError`? Or is it expected that not every subclass implements this?
(minor issue)